### DISCUSSION TITLE
Does using nchat make messages vulnerable?

### DISCUSSION THREAD

**Discussion by arcxne** [d99kris/nchat#166] [Q&A]
I absolutely love nchat 🤩 but I just wanted to know if using nchat makes all my chats and messages more vulnerable. Thank you in advance!

**Comment by arcxne** [comment]
Whoops, I totally missed the Security section in the Readme. For those wondering, everything is stored locally, so it is advised to use disk encryption.

Sorry for the trouble!

**Reply by d99kris** [reply]
Thanks for answering your own question 🙂 
Yep, you got it right, currently most of the nchat data is stored unencrypted.
